src/adaptors/moola/index.js

```diff
@@ -0,0 +1,59 @@
+const utils = require('../utils');
+
+
+const poolsFunction = async () => {
+    const apyData = await utils.getData(
+        'https://v2-srv-data-frm-smrt-cntract.herokuapp.com/get/getReserveData'
+    );
+
+    //Pools return stable APR
+
```

Comment:
why not just loop over the apyData, would make this all a bit cleaner

Reply:
Yeah, that was quick & dirty, i confess ^^'